dashboard/app/models/script_level.rb

```diff
@@ -573,6 +587,12 @@ def summarize_for_teacher_panel(student)
       navigator = navigator_info[0] if navigator_info
     end
 
+    if teacher.present?
+      # feedback for contained level is stored with the level ID not the contained level ID
```

Comment:
Oof, this is unfortunate.  Is there a unit test that makes sure we handle contained levels correctly (i.e. exercises the ternary on line 592)?

Reply:
No, I'll go ahead and add one